content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -85,6 +87,37 @@ The .NET Tracer works on .NET Core 2.0, 2.1, 2.2, and 3.0, but these versions re
 | JIT Compiler bug on Linux/x64                 | 2.0.x,</br>2.1.0-2.1.11,</br>2.2.0-2.2.5  | Upgrade .NET Core to the latest patch version, or follow steps in the linked issue | [DataDog/dd-trace-dotnet/issues/302][6] |
 | Resource lookup bug with a non `en-US` locale | 2.0.0                                     | Upgrade .NET Core to 2.0.3 or above                                    | [dotnet/runtime/issues/23938][7]        |
 
+## Supported Datadog Agent versions
+
+| **Datadog Agent version**   | **Package version** |
+|-----------------------------|---------------------|
+| [7.x][8]                   | Latest              |
+| [6.x][8]                   | Latest              |
+| [5.x][9]                   | Latest              |
+
+## Runtime support policy for .NET Core APM
+
+Datadog APM for .NET Core is built upon dependencies defined in specific versions of the host operating system, .NET Core runtime, certain .NET Core libraries, and the Datadog Agent/API. When these versions are no longer supported by their maintainers, Datadog APM for .NET Core limits its support for these as well.
```

Comment:
```suggestion
Datadog APM for .NET Core is built upon dependencies defined in specific versions of the host operating system, .NET Core runtime, certain .NET Core libraries, and the Datadog Agent/API. When a version of ??? is no longer supported by its dependencies, Datadog APM for .NET Core limits its support for ??? as well.
```
Second sentence needs rewording to be more precise. When I read the original version, I couldn't tell what the words "these versions" or "these" refer to.